aio/deploy/helm-chart/kubernetes-dashboard/README.md

```diff
@@ -0,0 +1,132 @@
+`ingress.annotations`               | Specify ingress class                                                                                                       | `kubernetes.io/ingress.class: nginx`
+`ingress.enabled`                   | Enable ingress controller resource                                                                                          | `false`
+`ingress.paths`                     | Paths to match against incoming requests. Both `/` and `/*` are required to work on gce ingress.                            | `[/]`
+`ingress.hosts`                     | Dashboard Hostnames                                                                                                         | `nil`
+`ingress.tls`                       | Ingress TLS configuration                                                                                                    | `[]`
+`resources`                         | Pod resource requests & limits                                                                                              | `limits: {cpu: 100m, memory: 100Mi}, requests: {cpu: 100m, memory: 100Mi}`
+`rbac.create`                       | Create & use RBAC resources                                                                                                 | `true`
+`rbac.clusterReadOnlyRole`          | If set, an additional role will be created with read only permissions to all resources listed inside.                       | `false`
+`serviceAccount.create`             | Whether a new service account name that the agent will use should be created.                                               | `true`
+`serviceAccount.name`               | Service account to be used. If not set and serviceAccount.create is `true` a name is generated using the fullname template. |
+`livenessProbe.initialDelaySeconds` | Number of seconds to wait before sending first probe                                                                         | `30`
+`livenessProbe.timeoutSeconds`      | Number of seconds to wait for probe response                                                                                | `30`
+`podDisruptionBudget.enabled`       | Create a PodDisruptionBudget                                                                                                | `false`
+`podDisruptionBudget.minAvailable`  | Minimum available instances; ignored if there is no PodDisruptionBudget                                                     |
+`podDisruptionBudget.maxUnavailable`| Maximum unavailable instances; ignored if there is no PodDisruptionBudget                                                   |
+`securityContext`                   | PodSecurityContext for pod level securityContext                                                                            | `{allowPrivilegeEscalation:false, readOnlyRootFilesystem: true, runAsUser: 1001, runAsGroup: 2001}`
```

Comment:
Might want to update this as it no longer reflects the contents of `values.yaml`

Reply:
done! Thanks.